app/views/state_file/questions/nj_tenant_eligibility/edit.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= form_with model: @form, url: { action: :update }, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card form-card--long" } do |f| %>
     <h1 class="h2"><%= title %></h1>
     <fieldset>
-      <legend style="margin-bottom: 2.5rem"><%= t(".label") %></legend>
+      <%= t(".label_html") %>
```

Comment:
[pebble] For semantic elements like legend, which must be a direct descendant of fieldset, having them live in the erb file would make it easier to review for accessibility

Reply:
[pebble] Relatedly, I'm not seeing a label_html in the en.yml file that has a legend, so it seems maybe this was just an accidental removal?

Reply:
Oops you're right - meant to remove the style, somehow accidentally removed the element